proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -36,6 +36,15 @@ message ModuleAccount {
   repeated string permissions  = 3;
 }
 
+// ModuleCredential represents a unclaimable pubkey for base accounts
+//
+// Since: cosmos-sdk 0.47
+message ModuleCredential {
+  string module_name        = 1;
+  bytes  module_account_key = 2;
```

Comment:
```suggestion
  bytes  address = 2;
```

Reply:
We derive the address when calling `.Address()`. Do you mean the module address?